virtualmachine/vsphere/vm.go

```diff
@@ -429,6 +429,72 @@ func (vm *VM) GetName() string {
 	return vm.Name
 }
 
+func (vm *VM) AddDisk() (err error) {
```

Comment:
specifying return value as variables or type only is sufficient. Prefer is type only.
`func (vm *VM) AddDisk() error`